Tests/MustacheTests/TemplateParserTests.swift

```diff
@@ -65,11 +65,7 @@ extension MustacheTemplate {
     }
 }
 
-#if hasFeature(RetroactiveAttribute)
-extension MustacheTemplate: @retroactive Equatable {}
-#else
```

Comment:
> 'retroactive' attribute does not apply; 'MustacheTemplate' is declared in the same package

Swift 6 mode does not seem unnecessary `@retroactive` in testable imported package.